### PR TITLE
Create 2015-05-20-gallery-1

### PR DESCRIPTION
**Pull request by aneda**
Is this along the lines of what you had in mind?  Isn't the code too long?

### CODE REVIEW

**Comment by aneda** [comment]
@chriddyp gallery example for line & scatter

**Comment by chriddyp** [comment]
love this! a few minor comments, and then feel free to merge!

File: _posts/python/line_and_scatter/2015-05-20-gallery-1 [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+# Find your api_key here: https://plot.ly/settings/api
+
+import plotly.plotly as py
+from plotly.graph_objs import *
+
+title = 'Main Source for News'
+
+labels = ['Television', 'Newspaper', 'Internet', 'Radio']
+
+colors = ['rgba(37,37,37,1)', 'rgba(115,115,115,1)', 'rgba(49,130,189, 1)', 'rgba(189,189,189,1)']
```
Comment (chriddyp) [inline _posts/python/line_and_scatter/2015-05-20-gallery-1]: I still find the black kinda harsh, could we switch to `rgb(67, 67, 67)`?

File: _posts/python/line_and_scatter/2015-05-20-gallery-1 [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    annotations.append(Annotation(xref ='paper',x = 0.05, y = y_trace[0], xanchor ='right', yanchor ='middle',
+        text =  label + ' {}%'.format(y_trace[0]), font = Font(family = 'Arial', size = 16, color = colors,),
+        showarrow = False,))
+    # labeling the right_side of the plot
+    annotations.append(Annotation(xref = 'paper', x = 0.95, y = y_trace[11], xanchor ='left', yanchor ='middle',
+        text = '{}%'.format(y_trace[0]), font = Font(family = 'Arial', size = 16, color = colors,), 
+        showarrow = False,))   
+        
+# Title
+annotations.append(Annotation(xref ='paper', yref = 'paper', x = 0.15, y = 1.05 , xanchor='center', yanchor='bottom',
```
Comment (chriddyp) [inline _posts/python/line_and_scatter/2015-05-20-gallery-1]: could we left-align the title with the x-axis? `xanchor='left'`, `x=0`?

File: _posts/python/line_and_scatter/2015-05-20-gallery-1 [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                
+# Source
+annotations.append(Annotation(xref ='paper', yref = 'paper', x = 0.5, y = -0.1 , xanchor='center', yanchor='top',
+    text = 'Source: PewResearch Center & Storytelling with data', font = Font(family = 'Arial', size = 12, 
+    color = 'rgb(150,150,150)'), showarrow = False,))    
+
+layout['annotations'] = annotations
+
+fig = Figure(data = traces, layout = layout)    
+plot_url = py.plot(fig, filename='news-soucre')
```
Comment (chriddyp) [inline _posts/python/line_and_scatter/2015-05-20-gallery-1]: typo! "news-source" :)

File: _posts/python/line_and_scatter/2015-05-20-gallery-1 [hunk 4]
```diff
@@ -0,0 +1,124 @@
+---
+name: How to Highlight the Particular Point
```
Comment (chriddyp) [inline _posts/python/line_and_scatter/2015-05-20-gallery-1]: maybe "Labelling Lines with Annotations" or "Minimal Line Chart" or "Minimal Line Chart<br>with Annotation Labels"?

**Pull request merged**